diff --git a/node_modules/playwright-core/lib/coreBundle.js b/node_modules/playwright-core/lib/coreBundle.js
index 9749434..fe4d2b7 100644
--- a/node_modules/playwright-core/lib/coreBundle.js
+++ b/node_modules/playwright-core/lib/coreBundle.js
@@ -25525,9 +25525,9 @@ var init_tracing = __esm({
           params: {
             error: serializeError(pageError.error),
             location: {
-              url: pageError.location.url,
-              line: pageError.location.lineNumber,
-              column: pageError.location.columnNumber
+              url: (pageError.location?.url ?? ""),
+              line: (pageError.location?.lineNumber ?? 0),
+              column: (pageError.location?.columnNumber ?? 0)
             }
           },
           pageId: page.guid
@@ -53882,9 +53882,9 @@ var init_browserContextDispatcher = __esm({
             error: serializeError(pageError.error),
             page: PageDispatcher.from(this, page),
             location: {
-              url: pageError.location.url,
-              line: pageError.location.lineNumber,
-              column: pageError.location.columnNumber
+              url: (pageError.location?.url ?? ""),
+              line: (pageError.location?.lineNumber ?? 0),
+              column: (pageError.location?.columnNumber ?? 0)
             }
           });
         });
